main/tmux/APKBUILD

```diff
@@ -15,10 +15,10 @@ source="https://github.com/tmux/tmux/releases/download/$pkgver/$pkgname-$pkgver.
 
 builddir="$srcdir/"$pkgname-$pkgver
 prepare() {
-	default_prepare || return 1
+	default_prepare
 
 	cd "$builddir"
-	aclocal && autoconf && automake --add-missing --force || return 1
+	aclocal && autoconf && automake --add-missing --force
```

Comment:
It was introduced in [661e85085ada6b6d1fc90742df5475ab64e9aa60](https://github.com/alpinelinux/aports/commit/661e85085ada6b6d1fc90742df5475ab64e9aa60).
Apparently it could be removed.